packages/components/src/components/hds/app-header/home-link.ts

```diff
@@ -12,21 +12,34 @@ import type { HdsInteractiveSignature } from '../interactive/';
 export interface HdsAppHeaderHomeLinkSignature {
   Args: HdsInteractiveSignature['Args'] & {
     icon: HdsIconSignature['Args']['name'];
+    isIconOnly?: boolean;
     color?: string;
-    ariaLabel: string;
+    text: string;
   };
   Element: HdsInteractiveSignature['Element'];
 }
 
 export default class HdsAppHeaderHomeLink extends Component<HdsAppHeaderHomeLinkSignature> {
-  get ariaLabel(): string {
-    const { ariaLabel } = this.args;
+  get text(): string {
+    const { text } = this.args;
 
     assert(
-      '@ariaLabel for "Hds::AppHeader::HomeLink" ("Logo") must have a valid value',
-      ariaLabel !== undefined
+      '@text for "Hds::AppHeader::HomeLink" must have a valid value',
+      text !== undefined
     );
 
-    return ariaLabel;
+    return text;
+  }
+
+  get icon(): HdsIconSignature['Args']['name'] {
+    const { icon } = this.args;
+
+    assert('@icon name must be provided', icon !== undefined);
```

Comment:
[issue] as a consumer, I see this error in the devtool console, and I don't know which component threw the error or where is coming from

Reply:
I've added a more explicit assertion for this